docs/extensibility/theme-support.md

```diff
@@ -278,3 +278,19 @@ Core blocks include default styles. The styles are enqueued for editing but are
 ```php
 add_theme_support( 'wp-block-styles' );
 ```
+
+## Responsive embedded content
+
+The embed blocks automatically apply styles to embedded content to reflect the aspect ratio of content that is embedded in an iFrame. A block styled with the aspect ratio responsive styles would look like:
```

Comment:
I found "aspect ratio responsive styles" a bit awkward to parse 🤷‍♂️, but then I couldn't think of anything else. 😅

Reply:
I've said and read them so much they've lost all meaning. "Embed" too. :rofl: